tillerinobot/src/main/java/tillerino/tillerinobot/lang/Indonesian.java

```diff
@@ -0,0 +1,271 @@
+				+ " Tillerino asli mengatakan bahwa ini tidak perlu dikhawatirkan, dan kita harus mencoba lagi."
+				+ " Jika kau mencemaskan ini, kau bisa [https://github.com/Tillerino/Tillerinobot/wiki/Contact memberitahunya] tentang ini. (reference "
+				+ marker + ")";
+	}
+
+	@Override
+	public String noInformationForModsShort() {
+		return "Tidak ada data untuk mod yang diminta.";
+	}
+
+	@Override
+	public Response welcomeUser(OsuApiUser apiUser, long inactiveTime) {
+		if (inactiveTime < 60 * 1000) {
+			return new Message("beep boop");
+		} else if (inactiveTime < 24 * 60 * 60 * 1000) {
+			return new Message("Selamat datang kembali, " + apiUser.getUserName() + ".");
+		} else if (inactiveTime > 7l * 24 * 60 * 60 * 1000) {
+			return new Message(apiUser.getUserName() + "...")
+				.then(new Message("...apakah itu kau? Sudah lama sekali!"))
+				.then(new Message("Senang bisa menemuimu kembali. Bisakah saya menarik minat anda dalam sebuah rekomendasi?"));
+		} else {
+			String[] messages = {
+					"kau terlihat seperti kau ingin sebuah rekomendasi.",
+					"sangat senang bisa menemuimu! :)",
+					"manusia favoritku. (Jangan beritahu manusia lain!)",
+					"kejutan yang ramah! ^.^",
+					"saya berharap kau akan datang. Semua manusia lain bodoh, tapi jangan beritahu yang lain bahwa saya berkata seperti itu! :3",
+					"apa yang anda rasa ingin anda lakukan hari ini?",
+			};
+
+			Random random = new Random();
+
+			String message = messages[random.nextInt(messages.length)];
+
+			return new Message(apiUser.getUserName() + ", " + message);
+		}
+	}
+
+	@Override
+	public String unknownCommand(String command) {
+		return "Perintah tidak diketahui \"" + command
+				+ "\". Ketik !help jika kau butuh bantuan!";
+	}
+
+	@Override
+	public String noInformationForMods() {
+		return "Maaf, Saya tidak bisa menyediakan informasi untuk mod itu saat ini.";
+	}
+
+	@Override
+	public String malformattedMods(String mods) {
+		return "Mod itu tidak terlihat benar. Mod bisa dikombinasi antara DT HR HD HT EZ NC FL SO NF. Kombinasi tanpa spasi atau karakter spesial. Contoh: !with HDHR, !with DTEZ";
+	}
+
+	@Override
+	public String noLastSongInfo() {
+		return "Aku tak ingat memberimu rekomendasi lagu...";
+	}
+
+	@Override
+	public String tryWithMods() {
+		return "Coba map ini dengan beberapa mod!";
+	}
+
+	@Override
+	public String tryWithMods(List<Mods> mods) {
+		return "coba map ini dengan " + Mods.toShortNamesContinuous(mods) + "!";
+	}
+
+	/**
+	 * The user's IRC nick name could not be resolved to an osu user id. The
+	 * message should suggest to contact @Tillerinobot or /u/Tillerino.
+	 * 
+	 * @param exceptionMarker
+	 *            a marker to reference the created log entry. six or eight
+	 *            characters.
+	 * @param name
+	 *            the irc nick which could not be resolved
+	 * @return
+	 */
+	public String unresolvableName(String exceptionMarker, String name) {
```

Comment:
I kinda forgot to mention this :( but this message doesnt need translations since its impossible for the system to use this in another language for technical reasons, sorry :(
So basically from line 108 till 124 can go...